SlevomatCodingStandard/Sniffs/Classes/RequireOneLinePropertyDocCommentSniff.php

```diff
@@ -0,0 +1,112 @@
+use SlevomatCodingStandard\Helpers\PropertyHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+class RequireOneLinePropertyDocCommentSniff implements \PHP_CodeSniffer\Sniffs\Sniff
+{
+
+	public const CODE_MULTI_LINE_PROPERTY_COMMENT = 'MultiLinePropertyComment';
+
+	/**
+	 * @return int[]
+	 */
+	public function register(): array
+	{
+		return [T_VARIABLE];
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile
+	 * @param int $propertyPointer
+	 */
+	public function process(\PHP_CodeSniffer\Files\File $phpcsFile, $propertyPointer): void
+	{
+		$tokens = $phpcsFile->getTokens();
+
+		// not a property
+		if (!PropertyHelper::isProperty($phpcsFile, $propertyPointer)) {
+			return;
+		}
+
+		// only validate properties with comment
+		if (!DocCommentHelper::hasDocComment($phpcsFile, $propertyPointer)) {
+			return;
+		}
+
+		// only validate properties without description
+		if (DocCommentHelper::hasDocCommentDescription($phpcsFile, $propertyPointer)) {
+			return;
+		}
+
+		$docCommentStartPointer = DocCommentHelper::findDocCommentOpenToken($phpcsFile, $propertyPointer);
+		$docCommentEndPointer = $tokens[$docCommentStartPointer]['comment_closer'];
+		$lineDifference = $tokens[$docCommentEndPointer]['line'] - $tokens[$docCommentStartPointer]['line'];
+
+		// already one-line
+		if ($lineDifference === 0) {
+			return;
+		}
+
+		// ignore empty lines
+		for ($currentLinePointer = TokenHelper::findFirstTokenOnNextLine($phpcsFile, $docCommentStartPointer);
+			$currentLinePointer !== null && $currentLinePointer < $docCommentEndPointer;
+			$currentLinePointer = TokenHelper::findFirstTokenOnNextLine($phpcsFile, $currentLinePointer)) {
+			$startingPointer = TokenHelper::findNext($phpcsFile, [T_DOC_COMMENT_STAR, T_DOC_COMMENT_CLOSE_TAG], $currentLinePointer, $docCommentEndPointer);
+
+			if ($startingPointer === null || $tokens[$startingPointer]['code'] === T_DOC_COMMENT_CLOSE_TAG) {
+				break;
+			}
+
+			$nextEffectivePointer = TokenHelper::findNextExcluding($phpcsFile, [T_DOC_COMMENT_WHITESPACE], $startingPointer + 1, $docCommentEndPointer + 1);
+
+			if ($tokens[$currentLinePointer]['line'] === $tokens[$nextEffectivePointer]['line']) {
+				continue;
+			}
+
+			$lineDifference--;
+		}
+
+		// looks like a compound doc-comment
+		if ($lineDifference > 2) {
+			return;
+		}
+
+		$fix = $phpcsFile->addFixableError(
+			sprintf(
+				'Found multi-line comment for property %s with single line content, use one-line comment instead.',
+				PropertyHelper::getFullyQualifiedName($phpcsFile, $propertyPointer)
+			),
+			$docCommentStartPointer,
+			self::CODE_MULTI_LINE_PROPERTY_COMMENT
+		);
+
+		if (!$fix) {
+			return;
+		}
+
+		$contentStartPointer = TokenHelper::findNextExcluding($phpcsFile, [T_DOC_COMMENT_WHITESPACE, T_DOC_COMMENT_STAR], $docCommentStartPointer, $docCommentEndPointer);
```

Comment:
@Majkl578 I think, there's a bug here.

There should be `$docCommentStartPointer + 1`. Currently `$contentStartPointer` cointains `$docCommentStartPointer`.

That's why the fixer doesn't work.